src/test/features/eligibility/routes/mcol-eligibility.ts

```diff
@@ -0,0 +1,23 @@
+import { expect } from 'chai'
+import * as request from 'supertest'
+
+import { attachDefaultHooks } from '../../../routes/hooks'
+import '../../../routes/expectations'
+
+import { Paths } from 'eligibility/paths'
+import { app } from '../../../../main/app'
+
+const mcolEligibilityPage: string = Paths.mcolEligibilityPage.uri
```

Comment:
Can you rename it to `pagePath` for consistency?

Reply:
addressed in latest commits